pages/wielders/[type].tsx

```diff
@@ -211,6 +233,7 @@ export const getStaticProps = withStaticBase(async (context) => {
     commandDescription: getTerm("Skills/Command/Level3/Description", locale),
     and: getSiteTerm("And", locale),
     or: getSiteTerm("Or", locale),
+    startingSkills: getSiteTerm("StartingSkills", locale),
```

Comment:
I think you could remove L:208 (I can't add a line-comment there) :
`command: getTerm("Commanders/Details/CommanderStat/Command", locale),`